homeassistant/components/scene/fibaro.py

```diff
@@ -0,0 +1,39 @@
+"""
+Support for Fibaro scenes.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.fibaro/
+"""
+import logging
+
+from homeassistant.components.scene import (
+    Scene)
+from homeassistant.components.fibaro import (
+    FIBARO_CONTROLLER, FIBARO_DEVICES, FibaroDevice)
+
+DEPENDENCIES = ['fibaro']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
We could change this to `async_setup_platform` since we don't appear to do any I/O here.